src/u_mapinfo.c

```diff
@@ -758,6 +749,34 @@ void U_ParseMapInfo(boolean is_default, const char *buffer, size_t length)
       continue;
     }
 
+    // Set default level progression here to simplify the checks elsewhere.
+    // Doing this lets us skip all normal code for this if nothing has been defined.
+    if (parsed.endpic[0] && (strcmp(parsed.endpic, "-") != 0))
+    {
+      parsed.nextmap[0] = 0;
+    }
+    else if (!parsed.nextmap[0] && !parsed.endpic[0])
+    {
+      if (!strcasecmp(parsed.mapname, "MAP30"))
```

Comment:
Unfortunately not. If you add a UMAPINFO entry for ExM8 without specifying a `nextmap` or `endpic` field, you'll end up on map ExM9 instead of getting the episode ending. I added the check for MAP30 merely for consistency.

Reply:
Okay, thanks.